source/auth_n.rst

```diff
@@ -4647,14 +4612,14 @@ This code will remain valid for 300 seconds (5 minutes).
 
 Next, you must collect this code from the user.
 
-.. note::
-
-  The code has to be sent to the correct Challenge ``href``. If your application is stateless, you could include the Challenge ``href`` in a hidden field on your form. If your application has a session, then you will want to attach the Challenge ``href`` to that session.
-
 Once you have the code, you send it to the same Challenge you created above:
 
 .. only:: rest
 
+  .. note::
+
+    The code has to be sent to the correct Challenge ``href``. If your application is stateless, you could include the Challenge ``href`` in a hidden field on your form. If your application has a session, then you will want to attach the Challenge ``href`` to that session.
```

Comment:
Shouldn't this note be present in Java as well? I only see it in REST but not in java